extensions/kafka-connect/src/test/java/com/hazelcast/jet/kafka/connect/KafkaConnectRandomIntIntegrationTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2021 Hazelcast Inc.
```

Comment:
2023 in each file ;)

Reply:
fixed https://github.com/hazelcast/hazelcast/pull/23312/commits/4ac690fec4cf15f99c465e995c3278181d3c7d3c